examples/light-switch-app/silabs/include/AppEvent.h

```diff
@@ -38,6 +38,7 @@ struct AppEvent
     {
         struct
         {
+            uint8_t ButtonId;
```

Comment:
This isn't used anywhere - can we remove it?

Reply:
Removed it.